src/storage.ts

```diff
@@ -540,7 +540,7 @@ export class Storage extends Service {
    * @property {number} [timeout] The amount of time in milliseconds to wait per http request before timing out.
    * @property {object[]} [interceptors_] Array of custom request interceptors to be returned in the order they were assigned.
    * @property {string} [apiEndpoint = storage.google.com] The API endpoint of the service used to make requests.
-   * @property {boolean} [useAuthWithCustomEndpoint] Controls whether or not to use authentication when using a custom endpoint.
+   * @property {boolean} [useAuthWithCustomEndpoint = false] Controls whether or not to use authentication when using a custom endpoint.
```

Comment:
@ddelgrosso1 please confirm that this does actually default to false. (I couldn't find a default value that we set, and so an undefined boolean would evaluate to false. But please make sure I'm not missing anything)

Reply:
You are correct, this is defined as an optional boolean or undefined. By default it would be undefined which means we would not use auth with a custom endpoint.